lib/IAMClient.js

```diff
@@ -545,8 +545,9 @@ class IAMClient {
                 return callback(null, obj, res.statusCode, res.statusMessage);
             }
             // Load the error from errors(arsenal)
-            return callback(
-                errors[obj.ErrorResponse.Error.Code], null, res.statusCode);
+            const retErr = obj.ErrorResponse
```

Comment:
I just updated this. This is to handle an Arsenal error, for example,
```json
{"key": "InternalError", "code":500,"description":"We encountered an internal error. Please try again."}
```
I have added the key prop. in Vault so that it can be mapped back to Arsenal Error.